client/monart.go

```diff
@@ -86,6 +87,19 @@ func (s MonartClient) FetchSecrets(fetchReq FetchSecretRequest) ([]byte, error)
 	return sec, nil
 }
 
+func (s MonartClient) GetSecretList(cfg GetSecretListConfig) ([]google.SecretObject, error) {
+	path := fmt.Sprintf("/environment/%s/secrets", cfg.Environment)
+	req, err := http.NewRequest(http.MethodGet, s.serverURL(path), nil)
+	if err != nil {
+		return nil, fmt.Errorf("unable to build Get SecretList request: %w", err)
+	}
+	result := new(GetSecretListResponse)
```

Comment:
We should have a response type here and we should be building location here, `google/gcs.go` should only set the fields as is.